data/json/items/chemicals_and_resources.json

```diff
@@ -567,10 +567,9 @@
     "weight": "460 g",
     "bashing": 1,
     "ammo_type": "components",
-    "charges": 2,
     "phase": "liquid",
     "container": "bottle_glass",
-    "freezing_point": 25,
+    "//freezing_point": 25,
```

Comment:
What's the idea here? Isn't `freezing_point` a valid node?

Reply:
Freezing point is only valid for comestibles.  This is an ammo.